test/game_test.rb

```diff
@@ -226,6 +226,12 @@ def test_grouped_season_returns_array_of_games_grouped_by_season
     assert_equal 20, Game.all.count
   end
 
+# the code was changed and this test is no longer passing 
+  def test_it_returns_all_games_by_seasons_given_team_id
+    assert_equal ({20122013 => 2, 20142015 => 4}), Game.games_by_season(3)
+    assert_equal ({20122013 => 2}), Game.games_by_season(6)
+  end
+
```

Comment:
I'm not sure who wrote this method but something changed and now the test doesn't pass. Maybe I didn't fix the merge conflicts correct, but it seems important that before doing a pull request we should run all our tests and make sure everything is passing one last time. I wasn't sure how to fix it, so I left it with a comment.

Reply:
I'll take a look. thanks for the comment